App_test.js

```diff
@@ -0,0 +1,60 @@
+// import React, { useState } from 'react';
```

Comment:
what is this file?

Reply:
that's the React hooks that I commented out, should've removed it